packages/schematics/angular/application/index.ts

```diff
@@ -223,6 +223,36 @@ function addAppToWorkspaceFile(options: ApplicationOptions, workspace: Workspace
     // }
 
     workspace.projects[options.name] = project;
+
+    // tslint:disable-next-line:no-any
```

Comment:
Remove `any`, use `JsonObject` instead.

Reply:
Used `any` because it is what is done just above for `project`. Changed to `JsonObject` as request in last commit, but forced me to cast each property.